modules/tinymce/src/plugins/lists/main/ts/actions/Indendation.ts

```diff
@@ -9,9 +9,12 @@ import * as Selection from '../core/Selection';
 import { Indentation } from '../listmodel/Indentation';
 import { listIndentation } from '../listmodel/ListsIndendation';
 
+const getContentEditableElements = (elements: HTMLElement[]): SugarElement<HTMLElement>[] =>
```

Comment:
This name is rather confusing as it sounds like it's just looking for anything with `contenteditable` as an attribute... I'd suggest naming this `getEditableElements` instead since that's what you're looking for here.

Reply:
Sure thing